pkg/apis/storage/validation/validation_test.go

```diff
@@ -400,27 +400,6 @@ func TestVolumeAttachmentValidation(t *testing.T) {
 			t.Errorf("expected success: %v %v", volumeAttachment, errs)
 		}
 	}
-
-	migrationDisabledErrorCases := []storage.VolumeAttachment{
-		{
-			// InlineSpec specified with migration disabled
```

Comment:
actually, move this up to the `migrationDisabledSuccessCases`... a valid inlineSpec should validate

Reply:
That would lead to:
```
--- FAIL: TestVolumeAttachmentValidation (0.00s)
    validation_test.go:418: expected failure: {{ } {foo      0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] []  []} {myattacher {<nil> 0x2222b80} node} {false map[] <nil> <nil>}} []
```

Reply:
in the `migrationDisabledSuccessCases`, success is expected

Reply:
just move this testcase into `migrationDisabledSuccessCases`